<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteMsgForm</class>
 <widget class="QWidget" name="WriteMsgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/images/leaveMsgPics/writeMsg.png);
	
	color: rgb(85, 0, 0);
	font: 20pt &quot;隶书&quot;;
}
QPushButton:hover
{
	color: rgb(0, 0, 255);
	border-image: url(:/images/leaveMsgPics/writeMsg-hover.png);
	font: 20pt &quot;隶书&quot;;
}
QPushButton:pressed
{
	font: 20pt &quot;隶书&quot;;
	color: rgb(255, 0,0);
	
	border-image: url(:/images/leaveMsgPics/writeMsg-pressed.png);
}</string>
   </property>
   <property name="text">
    <string> 提交</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
	
	
	background-color: rgb(221, 245, 255);
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>匿名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/images/EmailImage/quit.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}
QPushButton:hover{

background-color: rgba(230,0,0,0.5);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>601</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 245, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>582</width>
      <height>330</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>330</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_face">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	
	border-image: url(:/images/facePictures/face.png);
}
QPushButton:hover
{
	
	border-image: url(:/images/facePictures/face-pressed.png);
}
QPushButton:pressed
{
	
	border-image: url(:/images/facePictures/face-hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_face</zorder>
  <zorder>textEdit</zorder>
  <zorder>pushButton_submit</zorder>
  <zorder>comboBox</zorder>
  <zorder>checkBox</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
